app/bitcoin/servers.js

```diff
@@ -0,0 +1,19 @@
+class BitcoinServerRequests {
+
+    getBalance(address) {
+        const response = this.makeRequest(`https://www.blockchain.info/address/${address}?format=json`);
+        if (response) {
+            const json = JSON.parse(response.body);
+            return json.final_balance;
+        }
+    }
+
+    makeRequest(url) {
+        const request = new XMLHttpRequest();
+        request.open('GET', url, false);
+        request.send();
```

Comment:
See good ol' w3schools: there's an "onreadystatechange" thing you need to listen for.
Alternatively, we could use a library like [request](https://github.com/request/request)